test/multiverse/suites/rails/active_support_logger_test.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require './app'
+
+if defined?(ActiveSupport::Logger)
+  class ActiveSupportLoggerTest < Minitest::Test
+    include MultiverseHelpers
+    setup_and_teardown_agent
+
+    def setup
+      @output = StringIO.new
+      @logger = Logger.new(@output)
+      @broadcasted_output = StringIO.new
+      @broadcasted_logger = ActiveSupport::Logger.new(@broadcasted_output)
+      @logger.extend ActiveSupport::Logger.broadcast(@broadcasted_logger)
+
+      @aggregator = NewRelic::Agent.agent.log_event_aggregator
+      @aggregator.reset!
+    end
+
+    def test_broadcasted_logger_marked_skip_instrumenting
+      assert @broadcasted_logger.instance_variable_get(:@skip_instrumenting)
+      assert_nil @logger.instance_variable_get(:@skip_instrumenting)
+    end
+
+    def test_logs_not_forwarded_by_broadcasted_logger
+      message = 'Can you hear me, Major Tom?'
```

Comment:
👨‍🎤 love it!

I love the (unofficial) sequel and its cover too:

https://www.youtube.com/watch?v=wO0A0XcWy88

https://www.youtube.com/watch?v=KyFe5m2MU9Q